resources/js/svir_viewer_d3_tree.js

```diff
@@ -86,26 +89,34 @@
         function createSpinner(id, weight, name, field) {
             pdTempSpinnerIds.push("spinner-"+id);
             $('#projectDefWeightDialog').dialog("open");
-            if (field === undefined) {
-                $('#projectDefWeightDialog').append(
-                    '<p><label for="spinner'+id+'">'+name+': </label><input id="spinner-'+id+'" name="spinner" value="'+weight+'"></p>');
-            } else {
-                $('#projectDefWeightDialog').append(
-                    '<p><label for="spinner'+id+'">'+name+'('+field+'): </label><input id="spinner-'+id+'" name="spinner" value="'+weight+'"></p>');
+            var content = '<p><label style="clear: left; float: left; width: 10em;" for="spinner'+id+'">'+name;
```

Comment:
The semicolon ends the javascript instruction. It looks correct to me. But I agree there's something weird here. I don't see the `</p>`. I must check this.